server_admin/topics/threat/auth-sessions-limit.adoc

```diff
@@ -1,14 +1,30 @@
 
-{project_name} creates an authentication session for each authentication request within a realm. This session is internally represented by `RootAuthenticationSession`. Each `RootAuthenticationSession` can have multiple authentication sessions for a given client. In a browser authentication flow, the browser session translates to `RootAuthenticationSession` while browser tabs translate to a collection of authentication sessions within the `RootAuthenticationSession`.  
+When a login page is opened for the first time in a web browser, {project_name} creates an object called authentication session that stores some useful information about the request.
+Whenever a new login page is opened from a different tab in the same browser, {project_name} creates a new record called authentication sub-session that is stored within the authentication session.
+Authentication requests can come from any type of clients e.g. Admin CLI. In that case a new authentication session is also created with one authentication sub-session.
+Please note that authentication sessions can be created also in other ways than via a browser flow. The text below is applicable regardless of the source flow.   
 
 NOTE: This section describes deployments that use the {jdgserver_name} provider for authentication sessions.
+
+Authentication session is internally stored as `RootAuthenticationSessionEntity`. Each `RootAuthenticationSessionEntity` can have multiple authentication sub-sessions stored within the
+`RootAuthenticationSessionEntity` as a collection of `AuthenticationSessionEntity` objects. {project_name} stores authentication sessions in a dedicated {jdgserver_name} cache.
+The number of `AuthenticationSessionEntity` per `RootAuthenticationSessionEntity` contributes to the size of each cache entry. Total memory footprint of authentication session cache is determined by 
+number of stored `RootAuthenticationSessionEntity` and by number of `AuthenticationSessionEntity` within each `RootAuthenticationSessionEntity`.
+
+The number of maintained `RootAuthenticationSessionEntity` objects corresponds to the number of unfinished login flows from the browser. To keep the number of `RootAuthenticationSessionEntity`
+under control, using an advanced firewall control to limit ingress network traffic is recommended.
+
  
-Higher memory usage and a network overload in a cluster may occur for deployments where there are many active `RootAuthenticationSession` with a lot of authentication sessions.
+Higher memory usage may occur for deployments where there are many active `RootAuthenticationSessionEntity` with a lot of `AuthenticationSessionEntity`.
+If the load balancer does not support or is not configured for link:{installguide_stickysessions_link}[session stickiness], the load over network in a cluster can
+increase significantly. The reason for this load is that each request that lands to a node which does not own the appropriate authentication session needs to retrieve
```

Comment:
Change "lands to a node which" to "lands on a node that"
Always use "that" when the remainder of the sentence qualifies the first part the sentence.
Use "which" when the remainder of the sentence provides additional information, but does not qualify the first part of the sentence.